### PR TITLE
Dynamic Help() response

### PR DESCRIPTION
**Pull request by bcrochet**
Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: certification/internal/policy/operator/scorecard_basic_check_spec.go [hunk 1]
```diff
@@ -47,6 +50,13 @@ func (p *ScorecardBasicSpecCheck) Metadata() certification.Metadata {
 }
 
 func (p *ScorecardBasicSpecCheck) Help() certification.HelpText {
+	if p.fatalError {
+		return certification.HelpText{
+			Message: "There was a fatal error while running operator-sdk scorecard tests. " +
+				"Please see the preflight log for details. If necessary, set logging to be more verbose.",
+			Suggestion: "Set wait time to a higher value if this is a consistent error.",
```
Comment (komish) [inline certification/internal/policy/operator/scorecard_basic_check_spec.go:57]: Do we want to suggest this? This just carries the implication that you'll hit this in timeout cases only. Up to you, feel free to resolve.

**Review by komish — APPROVED** [review]
/LGTM

File: certification/internal/policy/operator/scorecard_basic_check_spec.go (hunk 1 above)
Comment (bcrochet) [reply certification/internal/policy/operator/scorecard_basic_check_spec.go:57]: This was literally just a POC. There needs to be a way to better determine what actual error occurred. Or it needs to be more generic.

**Review by acornett21 — APPROVED** [review]

File: certification/internal/policy/operator/scorecard_basic_check_spec.go [hunk 2]
```diff
@@ -47,6 +50,13 @@ func (p *ScorecardBasicSpecCheck) Metadata() certification.Metadata {
 }
 
 func (p *ScorecardBasicSpecCheck) Help() certification.HelpText {
+	if p.fatalError {
+		return certification.HelpText{
+			Message: "There was a fatal error while running operator-sdk scorecard tests. " +
+				"Please see the preflight log for details. If necessary, set logging to be more verbose.",
+			Suggestion: "If the logs are showing a context timeout, try setting wait time to a higher value.",
```
Comment (acornett21) [inline certification/internal/policy/operator/scorecard_basic_check_spec.go:57]: I think we would need to expose `PFLT_SCORECARD_WAIT_TIME` before we can say this in the help. Right now the value is internally hardcoded.

**Review by komish — COMMENTED** [review]
/lgtm

**Review by komish — APPROVED** [review]
I'll hit the right sequence of buttons to approve this PR some day!

**Review by jomkz — APPROVED** [review]

**Pull request merged**